Assets/Scripts/vent.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class vent : MonoBehaviour
```

Comment:
what's this for ?

Reply:
There's 2 vents in level 1 with fans in them that I just rotated with a script. Its just a small detail we could get rid of it.